trillian/examples/deployment/kubernetes/README.md

```diff
@@ -34,10 +33,13 @@ Process
    1. The `log_id:` field to contain the `tree_id` from the tree you provisioned into
       Trillian.
    1. The `prefix:` to the URL path prefix where you want your log API to be served.
-   1. The `public_key:` and `private_key:` entries to your own keys.
+   1. The `public_key:` and `private_key:` entries to your
+      [own keys](../../../docs/ManualDeployment.md#key-generation).  (The
+      [`to_proto`](https://github.com/google/trillian-examples/gossip/testdata/to_proto)
+      utility can help with the conversion to protobuf format.)
 1. Run the [deploy.sh](deploy.sh) script, using the same `config.sh` file you
    used for your Trillian deployment:
-  `./deploy.sh ../../../../../trillian/examples/deployment/kubernetes/config.sh`
+  `./deploy.sh ${GOPATH}/src/github.com/google/trillian/examples/deployment/kubernetes/config.sh`
```

Comment:
apparently `GOPATH` can have multiple ':' separated elements... :/

Reply:
Reverted to ugliness